code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -3,12 +3,15 @@
 
 /obj/machinery/atmospherics/components
 	hide = FALSE
-
-	var/welded = FALSE //Used on pumps and scrubbers
+	///Check if the component should be welded (mainly used by scrubbers and vents)
+	var/welded = FALSE
```

Comment:
This isn't actually true, welded doesn't allow a component to be welded, it's the "has this been welded or not" var. I have no idea why it's on components as a base

Reply:
should be good like this?

Reply:
Yeet